RUNTIMEEXECUTOR_LIB set to NOTFOUND

I am using react-native v0.66.5. I just updated to react-native-v8@1.5.2 from react-native-v8@0.66.3-patch.1 (because I was getting a v8 runtime error the update from react-native v0.66.4 to v0.66.5).

I am now getting a build error from gradle:

```
* What went wrong:
Execution failed for task ':react-native-v8:generateJsonModelDebug'.
> ./node_modules/react-native-v8/android/CMakeLists.txt : C/C++ debug|armeabi-v7a : CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
  Please set them or make sure they are set and tested correctly in the CMake files:
  RUNTIMEEXECUTOR_LIB
      linked by target "v8executor" in directory ./node_modules/react-native-v8/android
```

Google searches seem to return nothing for `RUNTIMEEXECUTOR_LIB`, so I'm not sure where to go next.

--- a/.github/workflows/android.yml
+++ b/.github/workflows/android.yml
@@ -12,11 +12,11 @@ jobs:
     runs-on: macos-10.15
     strategy:
       matrix:
-        rn-version: ['0.70', '0.69']
+        rn-version: ['0.70']
         v8-android-variant:
           [v8-android-jit, v8-android-jit-nointl, v8-android, v8-android-nointl]
         include:
-          - rn-version: '0.68'
+          - rn-version: ['0.69', '0.68', '0.67', '0.66']
             v8-android-variant: v8-android-jit
 
     steps:

--- a/.github/workflows/android.yml
+++ b/.github/workflows/android.yml
@@ -9,7 +9,7 @@ on:
 
 jobs:
   e2e-test:
-    runs-on: macos-10.15
+    runs-on: macos-latest
     strategy:
       matrix:
         rn-version: ['0.70']

--- a/.github/workflows/android.yml
+++ b/.github/workflows/android.yml
@@ -16,7 +16,13 @@ jobs:
         v8-android-variant:
           [v8-android-jit, v8-android-jit-nointl, v8-android, v8-android-nointl]
         include:
-          - rn-version: ['0.69', '0.68', '0.67', '0.66']
+          - rn-version: '0.69'
+            v8-android-variant: v8-android-jit
+          - rn-version: '0.68'
+            v8-android-variant: v8-android-jit
+          - rn-version: '0.67'
+            v8-android-variant: v8-android-jit
+          - rn-version: '0.66'
             v8-android-variant: v8-android-jit
 
     steps:

--- a/android/CMakeLists.txt
+++ b/android/CMakeLists.txt
@@ -118,12 +118,16 @@ find_library(
   PATHS ${LIBRN_DIR}
   NO_CMAKE_FIND_ROOT_PATH
 )
-find_library(
-  RUNTIMEEXECUTOR_LIB
-  runtimeexecutor
-  PATHS ${LIBRN_DIR}
-  NO_CMAKE_FIND_ROOT_PATH
-)
+if(${REACT_NATIVE_TARGET_VERSION} GREATER_EQUAL 68)
+  find_library(
+    RUNTIMEEXECUTOR_LIB
+    runtimeexecutor
+    PATHS ${LIBRN_DIR}
+    NO_CMAKE_FIND_ROOT_PATH
+  )
+else()
+  set(RUNTIMEEXECUTOR_LIB "")
+endif()
 find_library(
   V8_ANDROID_LIB
   v8android

